pkg/templatelookup/modulestatusinfo.go

```diff
@@ -13,52 +13,52 @@ var (
 	ErrInvalidModuleInStatus = errors.New("invalid module entry in Kyma status")
 )
 
-type AvailableModule struct {
+type ModuleStatusInfo struct {
 	v1beta2.Module
-	Enabled         bool
+	IsEnabled       bool
 	ValidationError error
-	Unmanaged       bool
+	IsUnmanaged     bool
 }
 
-func (a AvailableModule) IsInstalledByVersion() bool {
+func (a ModuleStatusInfo) IsInstalledByVersion() bool {
 	return a.configuredWithVersionInSpec() || a.installedwithVersionInStatus()
 }
 
 // configuredWithVersionInSpec returns true if the Module is enabled in Spec using a specific version instead of a channel.
-func (a AvailableModule) configuredWithVersionInSpec() bool {
-	return a.Enabled && a.Version != "" && a.Channel == ""
+func (a ModuleStatusInfo) configuredWithVersionInSpec() bool {
+	return a.IsEnabled && a.Version != "" && a.Channel == ""
 }
 
 // installedwithVersionInStatus returns true if the Module installed using a specific version (instead of a channel) is reported in Status.
-func (a AvailableModule) installedwithVersionInStatus() bool {
-	return !a.Enabled && shared.NoneChannel.Equals(a.Channel) && a.Version != ""
+func (a ModuleStatusInfo) installedwithVersionInStatus() bool {
+	return !a.IsEnabled && shared.NoneChannel.Equals(a.Channel) && a.Version != ""
 }
 
-// FindAvailableModules returns a list of AvailableModule objects based on the Kyma CR Spec and Status.
-func FindAvailableModules(kyma *v1beta2.Kyma) []AvailableModule {
+// FetchModuleStatusInfo returns a list of ModuleStatusInfo objects based on the Kyma CR Spec and Status.
```

Comment:
I think here it would also make sense to explain in more detail what this does. E.g., 

```go
// FetchModuleStatusInfo returns a list of [...] objects containing information about modules referenced by the Kyma CR.
// This includes modules that are enabled in `.spec.modules[]` and modules that are not enabled in `.spec.modules[]` but still contain an entry in `.status.modules[]`.
```